### ISSUE TITLE
Request: Seed Checker (Sub-Window) needs Calculation Progress Bar

### ISSUE THREAD

**Issue by Th3Alic3** [Leanny/SeedSearcher#42]
![image](https://user-images.githubusercontent.com/57371173/87834532-86e6f800-c83f-11ea-9719-8a9b9519f66c.png)

Hey Leanny,

I noticed the main window does have a progress bar but the sub-window for the Seed Checker does not.

The "SEARCH" option in the Seed Checker does not have any kind of progress bar (or pop-up calculating alert) which on long checks causes Windows to think that the program is not responsive.

In a future build, would you add something to keep Windows from thinking the app has crashed?

**Comment by Leanny** [comment]
Yeah. For the next version I will make the window not freeze when searching (and the search cancelable as well). Thanks for the reminder
